cypress/support/generate.js

```diff
@@ -0,0 +1,15 @@
+import { faker } from '@faker-js/faker';
+
+function generateWorker() {
+  const username = faker.internet.userName();
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+  const email = `${username}@mail.com`;
+  const age = faker.number.bigInt({ max: 60 });
+  const salary = faker.number.bigInt({ min: 300, max: 6000 });
```

Comment:
The 'bigInt' method from 'faker' doesn't exist. If you want to generate a random number between 300 and 6000, you should use 'faker.datatype.number({ min: 300, max: 6000 });'.